### PR TITLE
Adding more custom domain supported items

### PR DESCRIPTION
**Pull request by jeffreylees**
Adding more custom domains supported items + additional configuration details for the ones which require it.

https://auth0-docs-content-pr-6152.herokuapp.com/docs/custom-domains

https://auth0-docs-content-pr-6152.herokuapp.com/docs/custom-domains/additional-configuration

### CODE REVIEW

File: articles/custom-domains/index.md [hunk 1]
```diff
@@ -19,12 +19,19 @@ You'll need to register and own the domain name to which you're mapping your Aut
...
+* Database and social connections
+* Lock 11 with cross-origin authentication
+* Passwordless connections with Universal Login (The email link will be sent using the custom domain if the option is enabled in **Tenant Settings > Custom Domains**)
+* Google Apps connections
+* SAML connections and applications
+* WS-Fed clients (Auth0 as IDP aka WS-Fed Add-on)
+* Azure AD connections
+* ADFS connections
+* AD/LDAP connections (without Kerberos support)
+* AD/LAP connections (with Kerberos support)
```
Comment (mpaktiti) [inline articles/custom-domains/index.md]: typo: AD/LAP
Also why not merge these two in one bullet? `AD/LDAP connections (with or without Kerberos support)`

File: articles/custom-domains/additional-configuration.md [hunk 2]
```diff
@@ -103,6 +108,21 @@ If you want to use social identity providers with your custom domain, you must u
 ::: warning
 You cannot use [Auth0 developer keys](/connections/social/devkeys) with custom domains.
 :::
+  
+## Configure Google Apps connections
```
Comment (mpaktiti) [inline articles/custom-domains/additional-configuration.md:112]: This is all good but we must add this info also at the corresponding docs. For example, add this content at this doc https://auth0.com/docs/connections/enterprise/google-apps as an optional step if you are using custom domains.

File: articles/custom-domains/additional-configuration.md [hunk 3]
```diff
@@ -103,6 +108,21 @@ If you want to use social identity providers with your custom domain, you must u
...
+
+If you want to use Google Apps connections with your custom domain, you must update the Authorized redirect URI in your OAuth Client Settings. In the Google Developer Console, go to **Credentials**, choose your OAuth client in the list, and you will see a settings page with the app Client ID, secret, and other fields. In the **Authorized redirect URIs** field, add a URL in the format `https://<CUSTOM DOMAIN>/login/callback` that includes your custom domain (such as `https://login.northwind.com/login/callback`).
+
+## APIs
+
```
Comment (mpaktiti) [inline articles/custom-domains/additional-configuration.md:117]: This doc https://auth0.com/docs/api-auth/tutorials/verify-access-token should also have this info

File: articles/custom-domains/additional-configuration.md [hunk 4]
```diff
@@ -129,15 +149,22 @@ If you want to use SAML applications with your custom domain, you must update yo
...
-If you use Auth0 with a custom domain to issue Access Tokens for your APIs, then you must validate the JWT issuer(s) against your custom domains. For example, if you use the [express-jwt](https://github.com/auth0/express-jwt) middleware, you must do the following change.
+## Configure your WS-Fed Clients
 
-```js
-app.use(jwt({ 
-  issuer: 'https://YOUR-CUSTOM-DOMAIN',
-  //code omitted for brevity
-}));
-```
+If you want to use your WS-Fed applications with your custom domain with Auth0 as the IDP, you must update your Service Provider with new Identity Provider metadata from Auth0 (You can obtain the metadata reflecting the custom domain from: `https://<CUSTOM DOMAIN>/wsfed/FederationMetadata/2007-06/FederationMetadata.xml`).
```
Comment (mpaktiti) [inline articles/custom-domains/additional-configuration.md:156]: this doc https://auth0.com/docs/integrations/configure-wsfed-application should also have this info

File: articles/custom-domains/additional-configuration.md [hunk 5]
```diff
@@ -129,15 +149,22 @@ If you want to use SAML applications with your custom domain, you must update yo
...
 
-If you use the Auth0 APIs, such as the Management API, the API identifier will use your default tenant domain name (such as `https://${account.namespace}/userinfo` and `https://${account.namespace}/api/v2/`)
+## Configure Azure AD connections
+
```
Comment (mpaktiti) [inline articles/custom-domains/additional-configuration.md:159]: https://auth0.com/docs/connections/enterprise/azure-active-directory/v2#5-configure-reply-urls

File: articles/custom-domains/additional-configuration.md [hunk 6]
```diff
@@ -129,15 +149,22 @@ If you want to use SAML applications with your custom domain, you must update yo
...
+If you want to use Azure AD connections with your custom domain, you must update the Allowed Reply URL in your Azure AD settings. In your Azure Active Directory, go to **Apps registrations** and select your app. Then click **Settings -> Reply URLs** and add a URL in the format `https://<CUSTOM DOMAIN>/login/callback` that includes your custom domain (such as `https://login.northwind.com/login/callback`).
+
+## Configure ADFS connections
+
```
Comment (mpaktiti) [inline articles/custom-domains/additional-configuration.md:163]: https://auth0.com/docs/connections/enterprise/adfs

File: articles/custom-domains/additional-configuration.md [hunk 7]
```diff
@@ -129,15 +149,22 @@ If you want to use SAML applications with your custom domain, you must update yo
...
+The process is the same as [setting up the ADFS connection normally](/connections/enterprise/adfs) except that your callback URL needs to be changed from this format `https://<TENANT>.auth0.com/login/callback` to this one use `https://<YOUR-CUSTOM-DOMAIN>/login/callback`.
+
+## Configure AD/LAP connections
```
Comment (mpaktiti) [inline articles/custom-domains/additional-configuration.md]: typo: AD/LAP

File: articles/custom-domains/additional-configuration.md [hunk 8]
```diff
@@ -129,15 +149,22 @@ If you want to use SAML applications with your custom domain, you must update yo
...
+
+In order to use AD/LAP connections with Kerberos support and custom domains, you will need to update the ticket endpoint to work with the custom domain. [As mentioned in the [Auth0 AD/LAP connector documentation](/connector/modify#point-an-ad-ldap-connector-to-a-new-connection), the `config.json` file needs to be modified, with the `PROVISIONING_TICKET` value changed from this format `https://<TENANT>.auth0.com/p/ad/jUG0dN0R/info` to `https://<CUSTOM DOMAIN>/p/ad/jUG0dN0R/info`.
```
Comment (mpaktiti) [inline articles/custom-domains/additional-configuration.md]: typo: AD/LAP

Comment (mpaktiti) [reply articles/custom-domains/additional-configuration.md]: fix: `[As mentioned in the [Auth0 AD/LAP connector documentation](/connector/modify#point-an-ad-ldap-connector-to-a-new-connection)`

Comment (mpaktiti) [reply articles/custom-domains/additional-configuration.md]: This talks only about Kerberos support but on the other doc me also mention without Kerberos support. Is the process different or the same?

Comment (mpaktiti) [reply articles/custom-domains/additional-configuration.md]: add this info also at this doc https://auth0.com/docs/connector/modify#point-an-ad-ldap-connector-to-a-new-connection

**Review by mpaktiti — CHANGES_REQUESTED** [review]

**Comment by jeffreylees** [comment]
Thanks @mpaktiti I must have been tunnel visioned there.

Ready to check again, but also note that this PR will need to wait for @jhiner review which should be today/tomorrow I think.

File: articles/custom-domains/index.md [hunk 9]
```diff
@@ -19,12 +19,18 @@ You'll need to register and own the domain name to which you're mapping your Aut
...
 * Emails (the links included in the emails will use your custom domain)
-* Database and Social connections
-* Lock 11 with Cross Origin Authentication
-* SAML Connections and Applications
+* Database and social connections
+* Lock 11 with cross-origin authentication
+* Passwordless connections with Universal Login (The email link will be sent using the custom domain if the option is enabled in **Tenant Settings > Custom Domains**)
+* Google Apps connections
+* SAML connections and applications
+* WS-Fed clients (Auth0 as IDP aka WS-Fed Add-on)
```
Comment (jhiner) [inline articles/custom-domains/index.md]: I know I put this in the notes, but I think we shouldn't put "aka" like this. Maybe something cleaner / doc-friendly like "Auth0 as IDP using WS-Fed Add-on" ?

**Review by jhiner — CHANGES_REQUESTED** [review]

**Review by mpaktiti — APPROVED** [review]
LGTM

**Pull request merged**